### ISSUE TITLE
Add 1st gen LLMs to the index

### ISSUE THREAD

**Issue by liesenf** [Language-Technology-Assessment/main-database#22]
After @DBlankvoort brought this up, I do think that including 1st gen LLM in the index is a good and importantly feasible idea. This would allow us to broaden the index to all text generators instead of having to specify "instruct-tuned".

Major ones are:
BERT: actively marketed as OS already in 2018, in a way the original sin of openwashing.
https://research.google/blog/open-sourcing-bert-state-of-the-art-pre-training-for-natural-language-processing/

T5: cited as OS https://en.wikipedia.org/wiki/T5_(language_model)

**Comment by mdingemanse** [comment]
Agreed in principle though it's hard to know where to stop once the "instruction tuned" filter is off the table.

Do all fields make sense? Is there a useful distinction to be made between pre training and fine timing data? Is there an end model, distinct from a base model?

To do this well we must make sure our data model is up to the task
